.devops/pagopa-deploy-pipelines.yml

```
@@ -117,7 +117,7 @@ stages:
               azureSubscription: 'DEV-PAGOPA-SERVICE-CONN'
               scriptLocation: inlineScript
               inlineScript: |
-                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway --exclude-path ecommerce-fe
+                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway ecommerce-fe
```

```suggestion
                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway;ecommerce-fe
```
looking at [azcopy docs](https://learn.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-copy) multiple paths have to be separated by semicolon

---

Set in this way while you were posting :)